ioctl/newcmd/node/nodedelegate.go

```diff
@@ -280,3 +280,17 @@ func (m *delegatesMessage) String() string {
 	}
 	return output.FormatString(output.Result, m)
 }
+
+func getProbationList(client ioctl.Client, epochNum uint64) (*vote.ProbationList, error) {
```

Comment:
where is this private func used?

Reply:
here
https://github.com/iotexproject/iotex-core/blob/82bd08d5cf74727b3c5d5673b587fc7998f4fc12/ioctl/newcmd/node/nodeprobationlist.go#L54

Reply:
> here
> 
> https://github.com/iotexproject/iotex-core/blob/82bd08d5cf74727b3c5d5673b587fc7998f4fc12/ioctl/newcmd/node/nodeprobationlist.go#L54

they could be placed in the same file